specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/applicationGateway.json

```diff
@@ -1272,6 +1565,10 @@
           "$ref": "./network.json#/definitions/SubResource",
           "description": "Reference to the PublicIP resource."
         },
+        "privateLinkConfiguration": {
+          "$ref": "./network.json#/definitions/SubResource",
```

Comment:
based on our discussion, should it be readonly? How to get the id of private link configuration?

Reply:
No this is not a readonly property. This is a reference to configuration property below

Reply:
Let me know if you want to discuss about this more.

Reply:
readonly means user cannot set them. If it's not readonly, could you modify the example files and add this field into examples?

Reply:
Done I have added the examples in ApplicationGatewayCreate and ApplicationGatewayGet. 
Please resolve this conversations if I am addressed your comment.

Reply:
Hi. I left a new comment about this question. Resolved this one.